src/components/portrait/portrait.stories.js

```diff
@@ -27,6 +27,14 @@ Default.story = {
   name: "default",
 };
 
+export const Icon = () => {
```

Comment:
why do we need this story here?
I think we can just render the same story in Docs (using `.mdx` format) instead of creating new story in that way

Reply:
I created this story to be able to write a cypress test against it. This (given my limited knowledge) seemed to not be possible with the default story.

Reply:
if we provide add `knob` to the Default we will be able to add / change icon in `default` Test story

Reply:
@DlgSHi Is there an existing example of how to do this in another component?